app/workers/find_a_candidate/populate_pool_worker.rb

```diff
@@ -0,0 +1,21 @@
+class FindACandidate::PopulatePoolWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :default
+
+  def perform
+    application_forms_eligible_for_pool = Pool::Candidates.new(providers: [])
```

Comment:
```suggestion
    application_forms_eligible_for_pool = Pool::Candidates.application_forms_in_the_pool.select(:id)                                              
```

I think we need to change this a bit.

This way we also check if the candidate opted in and their account is not blocked, etc.

https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/models/pool/candidates.rb#L40-L47

Reply:
Yes, I think we can improve the naming of these methods 👍🏻 I'm trying to clean them up in my next PR and see if they make more sense?